pkg/plugin/util/util.go

```diff
@@ -284,25 +286,31 @@ func IsObjectBlocked(item runtime.Unstructured) (bool, string, error) {
 	if err != nil {
 		return false, "", errors.Errorf("Could not translate item kind %s to CRD name", item.GetObjectKind())
 	}
-	if IsResourceBlocked(crdName) {
+
+	blockListConfigMap, err := cmd.RetrieveBlockListConfigMap()
+	if err != nil {
+		return false, "", errors.WithStack(err)
+	}
+	if IsResourceBlocked(crdName, blockListConfigMap) {
 		return true, crdName, nil
 	}
 	return false, crdName, nil
 }
 
-func GetResources() []string {
-	desiredResources := make([]string, len(constants.ResourcesToHandle)+len(constants.ResourcesToBlock))
+func GetResources(blockListConfigMap map[string]string) []string {
+	desiredResources := make([]string, len(constants.ResourcesToHandle)+len(blockListConfigMap))
```

Comment:
`blockListConfigMap` could be nil, add unit test for this scenario

Reply:
Line 40: check if error type is k8serrors.IsNotFound. 
If !k8serrors.IsNotFound, return error.
if k8serrors.IsNotFound, 
line 45: blockListConfigMap = make(map[string]string)
so in this case blockListConfigMap will be empty but not nil.